frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.tsx

```diff
@@ -0,0 +1,108 @@
+import { Flex, Icon, NavLink, Paper, ScrollArea, TextInput } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+interface SimpleDataPickerProps {
+  selectedEntity?: TableId;
+  options: Options[];
+  onClick: (option: any) => void;
+}
+
+interface Options {
+  id: number;
+  display_name: string;
+}
+
+const TEN_OPTIONS_HEIGHT = 10 * 33;
+
+export function SimpleDataPicker({
+  selectedEntity,
+  options,
+  onClick,
+}: SimpleDataPickerProps) {
+  const shouldShowSearchBar = options.length > 10;
+  const [searchText, setSearchText] = useState("");
+  function filterSearch(option: Options): boolean {
+    if (searchText) {
+      return normalizeString(option.display_name).includes(
+        normalizeString(searchText),
+      );
+    }
+
+    return true;
+  }
+
+  if (options.length === 0) {
+    return (
+      <Paper w={CONTAINER_WIDTH} p="80px 60px">
+        <EmptyState
+          message={t`To pick some data, you'll need to add some first`}
+          icon="database"
+        />
+      </Paper>
+    );
+  }
+
+  const displayOptions = options.filter(filterSearch);
+  return (
+    <Paper
+      component={Flex}
+      w={CONTAINER_WIDTH}
+      p="sm"
+      mih="200px"
+      direction="column"
+    >
+      {shouldShowSearchBar ? (
+        <TextInput
+          data-autofocus
+          type="search"
+          icon={<Icon name="search" size={16} />}
+          mb="sm"
+          placeholder={t`Search…`}
+          onChange={e => setSearchText(e.target.value ?? "")}
+        />
+      ) : (
+        /**
+         * Behave like Mantine 7's `FocusTrap.InitialFocus`.
+         *
+         * This component disable the automatic focus on the first focusable element
+         * when there is no search bar.
+         */
+        <span aria-hidden data-autofocus tabIndex={-1} />
+      )}
+      <Flex direction="column" justify="center" style={{ flex: 1 }}>
+        {displayOptions.length >= 1 ? (
+          // @ts-expect-error - I think the typing for ScrollArea.Autosize is wrong. This might be fixed in Mantine 7 */}
+          <ScrollArea.Autosize mah={TEN_OPTIONS_HEIGHT} type="auto" mb="auto">
+            {displayOptions.map(option => {
+              const isSelected = selectedEntity === option.id;
+              const iconColor = isSelected
+                ? "--mb-color-text-white"
+                : "--mb-color-icon-primary";
+              return (
+                <NavLink
+                  key={option.id}
+                  active={selectedEntity === option.id}
+                  icon={<Icon color={`var(${iconColor})`} name="table" />}
+                  label={option.display_name}
+                  onClick={() => onClick(option)}
+                  variant="default"
+                />
+              );
+            })}
+          </ScrollArea.Autosize>
+        ) : (
+          <EmptyState message={t`Nothing here`} />
```

Comment:
Use a similar empty state to an existing UI
![Screenshot 2025-02-04 at 6 57 28 PM](https://github.com/user-attachments/assets/3bd77faf-20ba-44e7-baf7-379763f6df55)

https://github.com/metabase/metabase/blob/feb7db2c06009cfd0c1de84f1f93fe3054545219/frontend/src/metabase/query_builder/components/DataSelector/saved-entity-picker/SavedEntityList.tsx#L33